### PR TITLE
Use price estimation

### PR DESCRIPTION
**Pull request by rodrigoalmeida94**
In addition to test query, to fetch estimated cost of mosaic.

### CODE REVIEW

File: utils/widgets.py [hunk 1]
```diff
@@ -409,16 +409,31 @@ def test_workflow(selected_blocks):
...
-                except (KeyError, IndexError):
-                    estimated_credits = 0
-
+            for i, test_job in enumerate(test_jobs):
+                estimation = self.workflow.estimate_job(parameters[i])
                 test_df = test_job.get_results_json(as_dataframe=True)
-                test_df["estimatedCredits"] = estimated_credits
+                test_df["minEstimatedCredits"] = sum(
+                    [
+                        estimation[block]["blockConsumption"]["credit"]["min"]
```
Comment (chrieke) [inline utils/widgets.py:417]: Does this work?

`test_df["minEstimatedCredits"] = sum([block["blockConsumption"]["credit"]["min"] + block["machineConsumption"]["credit"]["min"] for block in estimation.values()])`

**Review by chrieke — APPROVED** [review]
LGTM!

File: utils/widgets.py (hunk 1 above)
Comment (rodrigoalmeida94) [reply utils/widgets.py:417]: Much better! Yeah it does, thank you 😊

**Review by thaisbendixen — APPROVED** [review]
LGTM!

**Pull request merged**